lib/broadside/cli/gli_commands.rb

```diff
@@ -95,22 +95,23 @@ def add_instance_flag(cmd)
 
 desc 'Deploy your application.'
 command :deploy do |d|
+  add_target_flag(d)
```

Comment:
this flag should be in the subcommands.. otherwise i believe the input gets parsed differently. it will expect 
`broadside deploy -t $TARGET short --tag $TAG` instead